rpms/machine-config-daemon.yml

```diff
@@ -9,6 +9,4 @@ content:
     specfile: machine-config-daemon.spec
 name: machine-config-daemon
 owners:
-  - coreos-devel@redhat.com
-distgit:
-  branch: rhaos-4.2-rhel-8
```

Comment:
well... this is definitely a problem but i don't think it's the problem you're asking about